docs/scraping-receivers.md

```diff
@@ -0,0 +1,143 @@
+# Scraping metrics receivers
+
+Scraping metrics receivers collect predefined set of metrics from externals sources. Typically, the external source 
+is a monitored entity providing data about itself in some arbitrary format, scraping receivers parse the data and 
+translate it to the OpenTelemetry Pipeline Data. Examples of the scrapings metrics receivers:
+
+- [Host Metrics Receiver](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/receiver/hostmetricsreceiver)
+- [Redis Receiver](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/receiver/redisreceiver)
+
+## Defining emitted metrics 
+
+Each scraping receiver has `metadata.yaml` file that MUST define all the metrics emitted by the receiver. The
+file is being used to generate an API for metrics recording, user settings to customize the emitted metrics and user
+documentation. The file schema is defined in 
+https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/cmd/mdatagen/metric-metadata.yaml. 
+Defining a metric in `metadata.yaml` DOES NOT guarantee that the metric will always be produced by the receiver. In
+some cases it may be impossible to fetch particular metrics from a system in a particular state. If it's know that 
+the metric is reported only under certain conditions (e.g. a specific version) of the external source, the 
+condition MUST be mentioned in the metric description.
```

Comment:
Is there a format that's recommended for this? It would be great to be consistent

Reply:
I don't know what format we can prescribe here. Conditions can vary significantly. Do you want to add an example like "The metric is only emitted for Redis version >= 6"?

Reply:
Great idea! Submitted an issue for this improvement for later https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/15995

Reply:
I'll remove this requirement from the doc for now